product_category_company/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import product_category
```

Comment:
product_template file is not imported here but inside models dir  !? weird.

Reply:
Weird. Fixed and added a test for this.